### DISCUSSION TITLE
Add text-align in RichText editor

### DISCUSSION THREAD

**Discussion by amandine16** [payloadcms/payload#1218] [Roadmap]
Hello, Can and how we add an dropdown with different text align as text-center , text-left, text-right ? Which can align images too

Thanks a lots ! You are the best !!

**Comment by jmikrut** [comment]
Hey @amandine16 - welcome! You can totally do this. 

Because our Rich Text field is built on SlateJS, you can look at the SlateJS docs to figure out how to build custom elements. But I think that these really belong in the core rather than having you build them yourself.

Here is an example:
https://www.slatejs.org/examples/richtext

And here is the code that they used:
https://github.com/ianstormtaylor/slate/blob/main/site/examples/richtext.tsx

I'm gonna tag this as Planned, meaning that our team will build this at some point. But if you are up for making a PR to build these into the core before then, that would be great!

**Comment by christian-reichart** [comment]
+1 for this! I probably do not have the time in the next months to make a PR, but having this as part of the core would be awesome, since this is probably the first thing my editors will ask when using the richt text field. :D
